src/pkg/services/m365/m365.go

```diff
@@ -47,12 +51,12 @@ type UserNoInfo struct {
 func UsersCompat(ctx context.Context, acct account.Account) ([]*User, error) {
 	errs := fault.New(true)
 
-	users, err := Users(ctx, acct, errs)
+	usersResult, err := Users(ctx, acct, errs)
```

Comment:
suggestion for a more idiomatic golang approach: since this is a small codespace, you can use a shorthand var name like `us` to mean `userSlice`
```suggestion
	us, err := Users(ctx, acct, errs)
```